packages/shared/lib/core/network/tests/network.test.ts

```diff
@@ -2,23 +2,27 @@ import { ITokenMetadata } from '@core/wallet/interfaces'
 
 import { IAuth, IClientOptions, INetwork, INode } from '../interfaces'
 
-import { NetworkProtocol, NetworkType } from '../enums'
+import { NetworkId, NetworkProtocol, NetworkType, TokenSupply } from '../enums'
 
 import { checkNodeUrlValidity, cleanAuth, getDefaultClientOptions, getOfficialNodes, isOfficialNetwork } from '../utils'
 
 describe('File: network.ts', () => {
-    function _buildNode(url: string): INode {
-        return {
-            url,
-            disabled: false,
+    function _buildNode(url: string | undefined): INode | undefined {
+        if (!url) {
```

Comment:
Let's reverse the conditional here so `if (url) { ... } else { ... }`; slightly easier to reason

Reply:
I guess it's a test though... not _as_ important :sweat_smile: